In file included from /usr/include/time.h:11,
/usr/include/sys/reent.h:264: error: expected identifier before `*' token
/usr/include/sys/reent.h:264: error: `_fpos64_t' has not been declared
/usr/include/sys/reent.h:264: error: `_fpos64_t' declared as function returning a function
In file included from /usr/include/pango-1.0/pango/pango-utils.h:25,
/usr/include/stdio.h:53: error: `_fpos64_t' does not name a type
In file included from /usr/include/pango-1.0/pango/pango-utils.h:25,
/usr/include/stdio.h:210: error: `fpos_t' has not been declared
/usr/include/stdio.h:216: error: expected `,' or `...' before `*' token
/usr/include/stdio.h:379: error: `fpos_t' has not been declared
/usr/include/stdio.h:380: error: expected `,' or `...' before `*' token
/usr/include/stdio.h:524: error: `_fpos64_t' has not been declared
/usr/include/stdio.h:524: error: `_fpos64_t' has not been declared
/usr/include/stdio.h:529: error: `_fpos64_t' has not been declared
/usr/include/stdio.h:529: error: `_fpos64_t' has not been declared
In file included from /usr/lib/gcc/i686-pc-cygwin/4.3.4/include/c++/bits/char_traits.h:48,
/usr/lib/gcc/i686-pc-cygwin/4.3.4/include/c++/cstdio:101: error: `::fpos_t' has not been declared

--- origsrc/gnome-commander-1.2.8.5/src/dialogs/gnome-cmd-advrename-regex-dialog.cc	2010-01-17 04:54:17.000000000 -0600
+++ src/gnome-commander-1.2.8.5/src/dialogs/gnome-cmd-advrename-regex-dialog.cc	2010-05-17 00:28:20.416306100 -0500
@@ -19,6 +19,7 @@
 */
 
 #include <config.h>
+#include <stdio.h>
 #include <regex.h>
 
 #include "gnome-cmd-includes.h"
--- origsrc/gnome-commander-1.2.8.5/src/gnome-cmd-file-list.cc	2010-02-13 14:39:25.000000000 -0600
+++ src/gnome-commander-1.2.8.5/src/gnome-cmd-file-list.cc	2010-05-17 00:31:08.990948000 -0500
@@ -19,6 +19,7 @@
 */
 
 #include <config.h>
+#include <stdio.h>
 #include <regex.h>
 
 #include "gnome-cmd-includes.h"
